recipes/scala/src/main/scala/com/adobe/platform/ml/Evaluator.scala

```diff
@@ -0,0 +1,62 @@
+/*#####################################################################
+# ADOBE CONFIDENTIAL
+# ___________________
+#
+#  Copyright 2018 Adobe
+#  All Rights Reserved.
+#
+# NOTICE:  All information contained herein is, and remains
+# the property of Adobe and its suppliers, if any. The intellectual
+# and technical concepts contained herein are proprietary to Adobe
+# and its suppliers and are protected by all applicable intellectual
+# property laws, including trade secret and copyright laws.
+# Dissemination of this information or reproduction of this material
+# is strictly forbidden unless prior written permission is obtained
+# from Adobe.
+#####################################################################*/
+
+package com.adobe.platform.ml
+
+import java.util
+
+import com.adobe.platform.ml.sdk.MLEvaluator
+import com.adobe.platform.ml.sdk.MLMetric
+import com.adobe.platform.ml.config.ConfigProperties
+import org.apache.spark.ml.Transformer
+import org.apache.spark.sql.DataFrame
+import org.apache.spark.sql.functions._
+
+
+class Evaluator extends MLEvaluator {
+
+  override def split(configProperties:ConfigProperties, data: DataFrame): (DataFrame, DataFrame) = {
+    val dataSplit = data.randomSplit(Array(0.8, 0.2), seed = 11L)
+    (dataSplit(0), dataSplit(1))
+
+  }
+
+  def evaluate(configProperties: ConfigProperties, model: Transformer, dataFrame: DataFrame): util.ArrayList[MLMetric] = {
```

Comment:
Do we have any standard formatter?  This line extends pretty far out and may be brought in given shorter chars-per-line policy.

Reply:
Currently, have the setting at 120